tests/data-files/m_3211605_ne_11_060_20200415_20200730.xml

```diff
@@ -5,7 +5,7 @@
 			<citeinfo>
 				<origin>USDA-Farm Production and Conservation Business Center (FPAC-BC)</origin>
 				<pubdate>20200729</pubdate>
-				<title>m_3211605_ne_11_060_20200415.tif</title>

```

Comment:
was this intentionally removed?

Reply:
no, thanks. It was removed whenever testing the 2020 resource_desc from cog_href. 

Added back in. There was an additional file representing this scenario committed.